tartiflette/sdl/transformers/schema/argument.py

```diff
@@ -0,0 +1,53 @@
+from typing import Optional, Union
+
+from tartiflette.sdl.transformers.schema import Name, GraphQLType, \
+    GraphQLDefinition, GraphQLValue
+
+
+class GraphQLArgumentDefinition(GraphQLDefinition):
+    # __slots__ = (
```

Comment:
Is that comment is useful for the future?

Reply:
It's been removed